docker-compose.yml

```diff
@@ -26,7 +26,7 @@ services:
         networks:
             - localdev
     npm:
-        image: postgres-django
+        image: hedera-django
```

Comment:
When working on multiple projects, it's nice for the local images to include the project name.

Reply:
Great point.